frontend/src/modules/users/actions.js

```diff
@@ -73,19 +74,31 @@ export const listTable = ({ offset = 0, limit = DEFAULT_PAGINATION_LIMIT } = {},
     return res
 };
 
-export const listVersions = (id) => (dispatch, getState) => {
+export const listRevisions = (id) => (dispatch, getState) => {
     const user = getState().users.itemsByID[id];
     if (user.isFetching) return Promise.resolve();
     const meta = { id };
     return dispatch(
         networkAction(
-            LIST_VERSIONS,
-            api.users.listVersions(id, user?.versions?.next ?? { limit: 10000 }),
+            LIST_REVISIONS,
+            api.users.listRevisions(id, user?.revisions?.next ?? { limit: 100 }),
             { meta }
         )
     );
 }
 
+export const listVersions = (userId, revisionId) => (dispatch, getState) => {
+    const user = getState().users.itemsByID[userId];
+    if (user.isFetching) return Promise.resolve();
+    const meta = { id: userId };
+    return dispatch(
+        networkAction(
+            LIST_VERSIONS,
+            api.users.listVersions(userId, {revision__id: revisionId, limit: 10000}),
```

Comment:
Is it necessary to maintain this limit ? I would still go for something less extreme like 400-500 or does it still causes issues ?

Reply:
yes it is necessary. otherwise the limit by default is 100 (and most of Ariane's largest modifications contain about 4K Versions per Revision)

Reply:
cant they be loaded with "load more" 500 each time ? i think if someone really need to dig into a 4000 versions revision they can tolerate to click a few times. Most likely they do not need to list all versions to find what they need and since there is little info at the revision level it is likely not even the revision they are looking for (*jedi hand waving*).

Reply:
They could but I am not sure it is worth implementing it. It happens for like 3-4 revisions, among the 95 000 existing revisions up to now..